### PR TITLE
Fix/demo intro

### LINKED ISSUE #1290: Demo fails on Array_ persistence creation
Is this issue due to changes in Array persistence?

demo

### PR DESCRIPTION
**Pull request by georgehristov**

### CODE REVIEW

File: demos/tutorial/intro.php [hunk 1]
```diff
@@ -137,10 +137,13 @@ function init(): void {
 */
 
 session_start();
+
+$_SESSION['x'] = $_SESSION['x'] ?? [];
+
 $session = new \atk4\data\Persistence\Array_($_SESSION['x']);
```
Comment (mvorisek) [inline demos/tutorial/intro.php]: better `?? []` here.

File: demos/tutorial/intro.php [hunk 2]
```diff
@@ -137,10 +137,13 @@ function init(): void {
...
 
 $form = \atk4\ui\Form::addTo($app);
-$form->setModel(new DemoInvoice($session))
+$form->setModel(new \atk4\ui\demo\DemoInvoice($session))
```
Comment (mvorisek) [inline demos/tutorial/intro.php]: should be not needed

Comment (georgehristov) [reply demos/tutorial/intro.php]: It doesn't work without, most probably because code is eval'ed

Comment (mvorisek) [reply demos/tutorial/intro.php]: I see, eval code without namespace, absolute path is probably better... Please check all other eval code.

**Review by mvorisek — APPROVED** [review]
Seems good, I added save hook also for the 2nd Model... Revert if not needed.

**Pull request merged**